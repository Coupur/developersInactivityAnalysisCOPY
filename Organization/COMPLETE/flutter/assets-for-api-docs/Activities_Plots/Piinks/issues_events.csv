id,date,event,creator_login
2496391856,2019-07-19 16:40:18,review_requested,Piinks
2496392045,2019-07-19 16:40:22,review_requested,Piinks
2496424733,2019-07-19 16:54:57,referenced,Piinks
2496424734,2019-07-19 16:54:57,merged,Piinks
2496424745,2019-07-19 16:54:57,closed,Piinks
2462203899,2019-07-05 09:24:01,mentioned,Piinks
2462203900,2019-07-05 09:24:01,subscribed,Piinks
2482952258,2019-07-15 13:45:10,mentioned,Piinks
2482952260,2019-07-15 13:45:10,subscribed,Piinks
2395854650,2019-06-06 23:01:39,review_requested,Piinks
2395873585,2019-06-06 23:15:33,review_requested,Piinks
2395873588,2019-06-06 23:15:33,review_requested,Piinks
2395922885,2019-06-06 23:51:49,referenced,Piinks
2395922886,2019-06-06 23:51:49,merged,Piinks
2395922895,2019-06-06 23:51:49,closed,Piinks
2470073394,2019-07-09 17:07:53,head_ref_deleted,Piinks
2350480633,2019-05-17 21:20:54,mentioned,Piinks
2350480634,2019-05-17 21:20:54,subscribed,Piinks
2334055163,2019-05-10 19:44:55,review_requested,Piinks
2334055305,2019-05-10 19:45:00,review_requested,Piinks
2334233679,2019-05-10 21:11:12,referenced,Piinks
2334233683,2019-05-10 21:11:12,merged,Piinks
2334233691,2019-05-10 21:11:12,closed,Piinks
2312339402,2019-05-01 17:12:29,review_requested,Piinks
2312515344,2019-05-01 18:29:52,review_dismissed,Piinks
2312643194,2019-05-01 19:25:21,review_requested,Piinks
2312708347,2019-05-01 19:55:34,referenced,Piinks
2312708350,2019-05-01 19:55:34,merged,Piinks
2312708354,2019-05-01 19:55:34,closed,Piinks
2382347335,2019-05-31 23:21:29,closed,Piinks
2251231291,2019-04-03 21:54:54,review_requested,Piinks
2251250591,2019-04-03 22:05:09,referenced,Piinks
2251250592,2019-04-03 22:05:09,merged,Piinks
2251250596,2019-04-03 22:05:09,closed,Piinks
2251250856,2019-04-03 22:05:18,head_ref_deleted,Piinks
2247643531,2019-04-02 19:25:52,referenced,Piinks
2247675938,2019-04-02 19:37:59,review_requested,Piinks
2247685522,2019-04-02 19:41:44,referenced,Piinks
2247685523,2019-04-02 19:41:44,merged,Piinks
2247685527,2019-04-02 19:41:44,closed,Piinks
2217724102,2019-03-20 18:00:18,renamed,Piinks
2217735720,2019-03-20 18:04:22,renamed,Piinks
2218018954,2019-03-20 19:53:53,referenced,Piinks
2218018956,2019-03-20 19:53:53,merged,Piinks
2218018962,2019-03-20 19:53:53,closed,Piinks
2218019869,2019-03-20 19:54:17,head_ref_deleted,Piinks
2207705459,2019-03-15 21:52:53,renamed,Piinks
2207740032,2019-03-15 22:14:08,review_requested,Piinks
2215190302,2019-03-19 22:22:01,head_ref_force_pushed,Piinks
2215220398,2019-03-19 22:38:48,review_requested,Piinks
2215318781,2019-03-19 23:41:18,referenced,Piinks
2215318784,2019-03-19 23:41:18,merged,Piinks
2215318787,2019-03-19 23:41:18,closed,Piinks
2218035941,2019-03-20 20:00:41,head_ref_deleted,Piinks
2204298342,2019-03-14 17:52:08,renamed,Piinks
2204813301,2019-03-14 21:02:03,referenced,Piinks
2204813304,2019-03-14 21:02:03,merged,Piinks
2204813306,2019-03-14 21:02:03,closed,Piinks
2204817509,2019-03-14 21:03:41,mentioned,Piinks
2204817510,2019-03-14 21:03:41,subscribed,Piinks
2218034977,2019-03-20 20:00:21,head_ref_deleted,Piinks
2194386285,2019-03-11 15:52:10,renamed,Piinks
2195360014,2019-03-11 21:44:25,renamed,Piinks
2195398333,2019-03-11 22:00:33,review_requested,Piinks
2195411478,2019-03-11 22:06:26,review_dismissed,Piinks
2197758958,2019-03-12 16:41:41,referenced,Piinks
2197758963,2019-03-12 16:41:41,merged,Piinks
2197758972,2019-03-12 16:41:41,closed,Piinks
2218034566,2019-03-20 20:00:12,head_ref_deleted,Piinks
2247685580,2019-04-02 19:41:45,referenced,Piinks
2247685577,2019-04-02 19:41:45,referenced,Piinks
