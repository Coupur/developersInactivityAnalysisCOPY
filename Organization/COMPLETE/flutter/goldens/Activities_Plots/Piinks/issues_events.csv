id,date,event,creator_login
2386458823,2019-06-03 22:44:20,referenced,Piinks
2386458824,2019-06-03 22:44:20,merged,Piinks
2386458833,2019-06-03 22:44:21,closed,Piinks
2386471223,2019-06-03 22:52:29,referenced,Piinks
2470075435,2019-07-09 17:08:35,head_ref_deleted,Piinks
